### ISSUE TITLE
Minified 2.2.7 js fails to load

### ISSUE THREAD

**Issue by zarko-tg** [Ranks/emojione#399]
```
Uncaught SyntaxError: Invalid regular expression: /<object[^>]*>.*?</object>|<span[^>]*>.*?</span>|<(?:object|embed|svg|img|div|span|p|a)[^>]*>|((\uD83D\uDC68...

...|\u26C8|\u2757))/: Nothing to repeat
    at new RegExp (<anonymous>)
    at emojione (emojione.min.js:8)
    at emojione.min.js:8
```

**Comment by Guillaume-Duval** [comment]
I have the same issue.

You can fix it temporarily by putting the unminified version.

Even though version 2.2.7 is installed, the lib / js / emojione.min.js module file remains the old file.
We see noted above: / *! Emojione 28-11-2016 * /
While the new file is dated: / *! Emojione 02-12-2016 * /

**Comment by caseyahenson** [comment]
This issue is resolved with the release of 3.0.
